src/main/java/com/mylib/core/services/BookService.java

```diff
@@ -84,6 +58,16 @@ public List<BookDto> getBooksList(){
 		return booksDto;
 	}
 	
+	public List<BookDto> getBooksListTest(String authorName, String illustratorName, String editorName, String collectionName){
+		List<Book> books = this.bookRepository.getBooksListFromView(authorName, illustratorName, editorName, collectionName);
+		List<BookDto> booksDto = new ArrayList<>();
+		for(Book book: books) {
```

Comment:
Utiliser un stream plutôt que la boucle for. C'est plus pour apprendre à manipuler les stream que par soucis de performance ou bonne pratique